.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
 composer.phar
+.idea
```

Comment:
Please remove this change, you should set a global gitignore (`git config --global core.excludesfile '~/.gitignore')` to ignore files related to the IDE you use.

Reply:
Thanks